integration-tests/spring-boot-properties/README.md

```diff
@@ -0,0 +1,12 @@
+### Current differences with Spring Boot properties.
+
+1. Default values assigned with `=` operator are not supported.
+2. Getters and setters are needed for all fields, including nested classes.
```

Comment:
Are you sure about this?

Reply:
Yes. I tried the Spring Boot way where a config field of a pojo type has only a getter and I ended up getting Quarkus extension that the field doesn't have a setter